### PR TITLE
feat: alpine image with goose ready

### PR DESCRIPTION
**Pull request by AgAriel**
add an alpine image with goose binary ready

### CODE REVIEW

File: goose/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12.5 as builder
+
+RUN go get github.com/pressly/goose/cmd/goose
```
Comment (ferhatelmas) [inline goose/Dockerfile]: pull a specific release (2.6)

File: goose/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+WORKDIR /go/src/github.com/pressly/goose
+RUN CGO_ENABLED=0 go install -tags='no_mysql no_sqlite3' ./cmd/goose
+
+FROM alpine:3.9
+
+COPY --from=builder /go/bin/goose /bin/goose
```
Comment (ferhatelmas) [inline goose/Dockerfile]: line break

File: goose/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# goose
+
+> Minimal alpine docker image with 
+[goose](https://github.com/pressly/goose) binary ready
```
Comment (ferhatelmas) [inline goose/README.md]: mention only postgres supported

File: goose/Dockerfile [hunk 4]
```diff
@@ -1,10 +1,14 @@
 FROM golang:1.12.5 as builder
 
-RUN go get github.com/pressly/goose/cmd/goose
+RUN go get github.com/lib/pq \
+    github.com/mattn/go-sqlite3 \
+    github.com/pkg/errors \
+    github.com/pressly/goose/cmd/goose
```
Comment (ferhatelmas) [inline goose/Dockerfile]: `go get github.com/pressly/goose/cmd/goose@v2.6.0` directly

File: goose/README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
 # goose
 
 > Minimal alpine docker image with 
-[goose](https://github.com/pressly/goose) binary ready
\ No newline at end of file
+[goose](https://github.com/pressly/goose) binary ready (postgres-only supported)
```
Comment (ferhatelmas) [inline goose/README.md]: `only-postgres supported` or ``postgres-only

File: goose/Dockerfile (hunk 4 above)
Comment (AgAriel) [reply goose/Dockerfile]: It doesn't work from scratch `go: cannot use path@version syntax in GOPATH mode
`

Comment (ferhatelmas) [reply goose/Dockerfile]: yes, `GO111MODULE=on`

Comment (ferhatelmas) [reply goose/Dockerfile]: it will be on  by default in 1.13 so later we can remove.

**Comment by ferhatelmas** [comment]
merged to be improved later.

**Pull request merged**